doc/source/dev/contributor/rendering_documentation.rst

```diff
@@ -21,9 +21,9 @@ Rendering Documentation Locally
 To render the documentation on your own machine (macOS or Linux):
 
 0. Ensure that you have a working SciPy :ref:`dev-env` active.
-#. Install `Sphinx`_ or ensure that your installation is up to date. For
+#. Install `Sphinx`_ and `matplotlib`_. For
    example, if you're using the Anaconda distribution of Python, enter in a
-   terminal window ``conda install sphinx`` or ``conda update sphinx``.
+   terminal window ``conda install sphinx matplotlib``.
```

Comment:
you need numpydoc as well.

Reply:
Somehow I think it gets installed without listing explicitly, because I built the docs I'm the container (with `make html-scipyorg`) without explicitly installing it.

Reply:
I think it's managed with `git submodule`

Reply:
Indeed:
```
$ cat .gitmodules 
[submodule "doc/scipy-sphinx-theme"]
	path = doc/scipy-sphinx-theme
	url = https://github.com/scipy/scipy-sphinx-theme.git
[submodule "doc/sphinxext"]
	path = doc/sphinxext
	url = https://github.com/numpy/numpydoc.git
[submodule "doc/source/_static/scipy-mathjax"]
	path = doc/source/_static/scipy-mathjax
	url = https://github.com/scipy/scipy-mathjax.git
```

when you clone the repo and want to build the docs, you should do. I believe you get a message to that extent if you forget.
```
git submodule init
git submodule update
```

Reply:
Thanks. In gh-10724, @peterbell10 suggested simplifying:
`git submodule update --init`